CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [0.9.0]
 
+* Implement `*Py` traits for arrays (up to 32).
```

Comment:
Can you please change the line to read the following and also move it to the `Added` section of the `0.9.0` changelog.

```suggestion
* Implement `FromPyObject` and `IntoPy<PyObject>` traits for arrays (up to 32). [#778](https://github.com/PyO3/pyo3/pull/778)
```